src/main/java/com/atlassian/db/replica/internal/SqlQuery.java

```diff
@@ -0,0 +1,30 @@
+package com.atlassian.db.replica.internal;
+
+public class SqlQuery {
```

Comment:
[ Consider declaring the class as final](https://github.com/wyrzyk/java-api-design-checklist/blob/master/CHECKLIST.md#2314-consider-declaring-the-class-as-final)

Reply:
Ahh, it's internal ❤️ .

Reply:
Yes, but still I added it